docs/api-guide/routers.md

```diff
@@ -72,14 +72,21 @@ Alternatively you can use Django's `include` function, like so…
         url(r'^', include(router.urls)),
     ]
 
-Router URL patterns can also be namespaces.
+To include the router URL patterns with an application namespace pass a
+`(router.urls, 'app_name')` 2-tuple to `include`.
```

Comment:
I added the `''` around `app_name` here. Hopefully that works. 

The option is something like `(PATTERN_LIST, APP_NAME)`